README.md

```diff
@@ -0,0 +1,18 @@
+# meta-ci4rail-bsp
```

Comment:
Adopt whole README to meta-ci4rail-devtools

Reply:
ok